man/rgbasm.5

```diff
@@ -328,76 +328,100 @@ with a non-zero constant as either operand will be constant 1, even if the other
+The precision of a literal can be overridden by appending a
 .Dq q
-to it followed by the number of fractional bits, such as
-.Ql 12.34q8 .
-.Pp
-Since fixed-point values are still just integers, you can use them in normal integer expressions.
-Some integer operators like
-.Sq +
-and
-.Sq -
-don't care whether the operands are integers or fixed-point.
-You can easily truncate a fixed-point number into an integer by shifting it right by 16 bits.
-It follows that you can convert an integer to a fixed-point number by shifting it left.
-.Pp
-The following functions are designed to operate with fixed-point numbers:
+followed by the number of fractional bits:
+.Ql 1234.5q8
+is equal to $0004d280
 .EQ
 delim $$
 .EN
+($= 1234.5 * 2 sup 8$).
+.Pp
+The following functions are designed to operate with fixed-point numbers:
 .Bl -column -offset indent "ATAN2(y, x)"
 .It Sy Name Ta Sy Operation
-.It Fn DIV x y Ta Fixed-point division $( x \[di] y ) \[mu] ( 2 ^ precision )$
-.It Fn MUL x y Ta Fixed-point multiplication $( x \[mu] y ) \[di] ( 2 ^ precision )$
-.It Fn FMOD x y Ta Fixed-point modulo $( x % y ) \[di] ( 2 ^ precision )$
-.It Fn POW x y Ta $x$ to the $y$ power
-.It Fn LOG x y Ta Logarithm of $x$ to the base $y$
+.It Fn DIV x y Ta Fixed-point division
+.It Fn MUL x y Ta Fixed-point multiplication
+.It Fn FMOD x y Ta Fixed-point modulo
+.It Fn POW x y Ta $x sup y$
+.It Fn LOG x y Ta $log sub y ( x )$ (such that
+.Ic LOG Ns ( Ic POW Ns ( x , y ) , y ) == x)
 .It Fn ROUND x Ta Round $x$ to the nearest integer
-.It Fn CEIL x Ta Round $x$ up to an integer
-.It Fn FLOOR x Ta Round $x$ down to an integer
+.It Fn CEIL x Ta Round $x$ up to the nearest integer
+.It Fn FLOOR x Ta Round $x$ down to the nearest integer
 .It Fn SIN x Ta Sine of $x$
 .It Fn COS x Ta Cosine of $x$
 .It Fn TAN x Ta Tangent of $x$
-.It Fn ASIN x Ta Inverse sine of $x$
-.It Fn ACOS x Ta Inverse cosine of $x$
-.It Fn ATAN x Ta Inverse tangent of $x$
+.It Fn ASIN x Ta Inverse sine of $x$ (such that
+.Ic ASIN Ns ( Ic SIN Ns ( x ) ) == x)
+.It Fn ACOS x Ta Inverse cosine of $x$ (such that
+.Ic ACOS Ns ( Ic COS Ns ( x ) ) == x)
+.It Fn ATAN x Ta Inverse tangent of $x$ (such that
+.Ic ATAN Ns ( Ic TAN Ns ( x ) ) == x)
 .It Fn ATAN2 y x Ta Angle between $( x , y )$ and $( 1 , 0 )$
 .El
 .EQ
 delim off
 .EN
 .Pp
-All of these fixed-point functions can take an optional final argument, which is the precision to use.
+Operators not in the list above, such as
+.Sq +
+and
+.Sq - ,
+work correctly with fixed-point operands as well (but
+.Em both
+operands must be fixed-point: use
+.Ql 1.0 + 2.0 ,
+not
+.Ql 1.0 + 2 ;
+and use
+.Ql MUL(1.0, 2.0)
+or
+.Ql 1.0 * 2 ,
+not
+.Ql MUL(1.0, 2) ) .
+.Pp
+All of these fixed-point functions can take an optional final argument, which is the precision to use
+.Em for the operation .
 For example,
 .Ql MUL(6.0q8, 7.0q8, 8)
 will evaluate to
 .Ql 42.0q8
 no matter what value is set as the current
 .Cm Q
 option.
+Currently,
```

Comment:
Everything in this doc is "current", and there's been no suggestion that this would change in future.
```suggestion
```

Reply:
I think it *would* be worth warning about.

Reply:
Then that can get its own issue, discussion, and maybe a PR in which this doc would be updated.

Reply:
Agreed; but, in the meantime, should this be kept?

Reply:
(We don't say "Currently these are all the functions that exist[, but we have ideas to Eventually let users define their own]"; or "Currently a charmap is a single byte[ but we've had a feature request for years to extend that]".)

Reply:
I'd prefer to drop *all* "currently" and the like, given [Google's guidance about timeless documentation](https://developers.google.com/style/timeless-documentation). When comparing of features between supported versions, use language like the notes in the Python library reference that begin with "Added in version 3.2" or "Changed in version 3.7".

Reply:
"Do it because Google does it" is the single worst argument I've seen in this thread, for any side :P

Reply:
As far as I know, we have never yet *had* any time-anchored language; "current" appears a lot but in regard to the state of RGBASM/LINK during processing.

Reply:
Then maybe it's a good idea not to start.

Reply:
Whoops, there is *one*:
> The options that OPT can modify are currently: b, g, p, Q, and r.

Reply:
I meant not "because consistency with Google style is desirable" but "for the reasons described in this article, which happens to be published by Google."